drivers/filesystems/fs_rec/ext.c

```diff
@@ -19,21 +19,21 @@
 
 BOOLEAN
 NTAPI
-FsRecIsExt2Volume(IN PEXT2_SUPER_BLOCK SuperBlock)
+FsRecIsExtVolume(IN PEXT_SUPER_BLOCK SuperBlock)
 {
     /* Just check for magic */
-    return (SuperBlock->Magic == EXT2_SUPER_MAGIC);
+    return (SuperBlock->Magic == EXT_SUPER_MAGIC);
```

Comment:
All the EXTn filesystems use the same magic?

Reply:
yes https://github.com/torvalds/linux/blob/master/include/uapi/linux/magic.h#L27